### DISCUSSION TITLE
miles finishing setup hanging (I read the thing hear me out)

### DISCUSSION THREAD

**Discussion by ShortShrimp** [small-cactus/M.I.L.E.S#29] [Q&A]
Hi, im on mac, somehow managed to finish setup but it isshowing me the 
 "Please wait for the downloads, then stay for a quick speech recognition test.
The app is not hanging, it just takes a minute to startup."
but it has been stuck like that for an hour, so im quite sure it is not working properly. please help 
thanks 

ps i have an m3 air and good wifi so it is probably not a performance problem

**Comment by ShortShrimp** [comment]
also, the start download button is still there, pressing it does nothing

**Comment by small-cactus** [comment]
Try manually running the config.py script, the download button just runs
that script in the background, you can manually run it instead.

**Comment by b3ta1911** [comment]
I have a similar problem, but on mac. I get this error. What do I need to do?
Starting the Node.js application...

> m.i.l.e.s@2.0.0 start
> electron .

Server started on http://localhost:3000
API keys are set, starting server and backend...
Server is already running on port 3000
Python Error: /Users/beta/Library/Python/3.9/lib/python/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(

Python Output: Tool list JSON has been saved to /Users/beta/Miles-env/M.I.L.E.S/Miles-V2/plugin_tool_list.json

Python Error: /Users/beta/Library/Python/3.9/lib/python/site-packages/pydub/utils.py:170: RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work
  warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)

Python Output: tflite_runtime is not available on macOS, using ONNX model.

Python Error: Traceback (most recent call last):
  File "/Users/beta/Miles-env/M.I.L.E.S/Miles-V2/main.py", line 1198, in <module>
    main()
  File "/Users/beta/Miles-env/M.I.L.E.S/Miles-V2/main.py", line 1128, in main
    owwModel = initialize_wake_word_model()
  File "/Users/beta/Miles-env/M.I.L.E.S/Miles-V2/main.py", line 1118, in initialize_wake_word_model
    owwModel = Model(wakeword_models=[MODEL_PATH], inference_framework=INFERENCE_FRAMEWORK)
  File "/Users/beta/Library/Python/3.9/lib/python/site-packages/openwakeword/utils.py", line 686, in wrapped
    return func(*args, **new_kwargs)
  File "/Users/beta/Library/Python/3.9/lib/python/site-packages/openwakeword/model.py", line 213, in __init__
    self.preprocessor = AudioFeatures(inference_framework=inference_framework, **kwargs)
  File "/Users/beta/Library/Python/3.9/lib/python/site-packages/openwakeword/utils.py", line 84, in __init__
    self.melspec_model = ort.InferenceSession(melspec_model_path, sess_options=sessionOptions,
  File "/Users/beta/Library/Python/3.9/lib/python/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 419, in __init__
    self._create_inference_session(providers, provider_options, disabled_optimizers)
  File "/Users/beta/Library/Python/3.9/lib/python/site-packages/onnxruntime/capi/onnxruntime_inference_collection.py", line 480, in _create_inference_session
    sess = C.InferenceSession(session_options, self._model_path, True, self._read_config_from_model)
onnxruntime.capi.onnxruntime_pybind11_state.NoSuchFile: [ONNXRuntimeError] : 3 : NO_SUCHFILE : Load model from /Users/beta/Library/Python/3.9/lib/python/site-packages/openwakeword/resources/models/melspectrogram.onnx failed:Load model /Users/beta/Library/Python/3.9/lib/python/site-packages/openwakeword/resources/models/melspectrogram.onnx failed. File doesn't exist

Python script exited with code: 1
Setup and launch completed.

and i runned both pip3 install and pip install on all requirements.
